test/1.2/legacy/suppress_WBIGUNROLL.dml

```diff
@@ -0,0 +1,80 @@
+/*
+  © 2024 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.2;
+
```

Comment:
I think we should make the warning specific to 1.4, big unroll is a valid idiom in 1.2 (as demonstrated by dml-builtins). (Unsure how common valid non-collapsed unrolls are in practice, though)

Reply:
The thing you're talking about is already, intentionally, not covered by `WBIGUNROLL`: this _only_ warns about big unrolls when performed on compile-time lists specified by the model, and does not warn about `foreach` on `ObjectList`.

Reply:
> (Unsure how common valid non-collapsed unrolls are in practice, though)
_Too common._ Don't you remember the context behind this PR's creation? And delay to merging?

Reply:
Discussed on IM, dropping this request.